packages/overlay/src/integrations/sentry/data/sentryStore.ts

```diff
@@ -410,6 +414,45 @@ const useSentryStore = create<SentryStoreState & SentryStoreActions>()((set, get
     );
   },
 
+  getAggregateCallData(): AggregateCallData[] {
+    const aggregateCalls = new Map<string, AggregateCallData>();
+    for (const [traceId, profile] of this.profilesByTraceId) {
+      for (let sampleIdx = 0; sampleIdx < profile.samples.length - 1; sampleIdx++) {
+        const sample = profile.samples[sampleIdx];
+        const nextSample = profile.samples[sampleIdx + 1];
+        // TODO: Handle the case where nextSample is undefined -- use the end of the profile or associated trace
+        const duration = nextSample.start_timestamp - sample.start_timestamp;
```

Comment:
```suggestion
        const duration = Math.max(0, nextSample.start_timestamp - sample.start_timestamp)
```

just a safety clamp in case of any issue in sync

Reply:
Don't think we should do this as samples should always be sorted. If not I'd prefer to know about that somehow rather than masking it with 0.